src/fe/inf_fe_map.C

```diff
@@ -203,11 +203,20 @@ Point InfFEMap::inverse_map (const unsigned int dim,
       libmesh_error_msg("Invalid dim = " << dim);
     }
 
+#ifdef DEBUG
+  // In debug mode, the validity of base_elems id is checked.
```

Comment:
This is a little surprising.  We do assert validity in DofObject::id() ... but where in inverse_map do we end up *querying* an id?  Just in the error messages when the mapping doesn't converge?

Reply:
And in any case, won't the same test end up happening in devel mode?  So instead of ifdef DEBUG we'd want ifndef NDEBUG?

Reply:
yes, it is queried only in case of non-convergence. But than the program stops; this might be anoying because if one point does not converge (e.g. in postprocessing), this is not a big issue...

Reply:
Oh, it's definitely an issue, and you put in the right fix; I'd just wanted to make sure I understood the issue.